fitsnap3lib/calculators/lammps_snap.py

```diff
@@ -388,6 +388,12 @@ def _collect_lammps(self):
             pt.shared_arrays['w'].array[index] = self._data["eweight"]
             pt.fitsnap_dict['Row_Type'][dindex:dindex + bik_rows] = ['Energy'] * nrows_energy
             pt.fitsnap_dict['Atom_I'][dindex:dindex + bik_rows] = [int(i) for i in range(nrows_energy)]
+            # create an atom types list for the energy rows, if bikflag=1
+            if config.sections['BISPECTRUM'].bikflag:
+                pt.fitsnap_dict['Atom_Type'][dindex:dindex + bik_rows] = [int(i) for i in lmp_types]
+            else:
+                pt.fitsnap_dict['Atom_Type'][dindex:dindex + bik_rows] = [0]
```

Comment:
This inserts the list `[0]` instead of the value `0`. So the array will look like this (for example)
`[list([0]) 1 1 ... 0 0 0]`
I tried
`pt.fitsnap_dict['Atom_Type'][dindex] = 0`
but it's not allowed because we need `len(value)==1` in parallel tools: https://github.com/rohskopf/FitSNAP/blob/749f8e4e38f4c4605b0a4c95d0bfad6ba2c07dbc/fitsnap3lib/parallel_tools.py#L731
I'm sure there's a good reason for that assert so I left it alone, but it requires us to specify start and end indices when inserting a length 1 list.